modules/las/worker-loader.js

```diff
@@ -1,5 +1,5 @@
 // The bundled worker is imported as an inline string
-import worker from '../dist/las-loader.worker.js';
+import worker from './dist/las-loader.worker.js';
```

Comment:
This file will not be transpiled and break any environment that does not support esm style `import`.